fluentui_drawer/src/main/res/layout/view_sheet_horizontal_item_view.xml

```diff
@@ -26,6 +26,7 @@
     <TextView
         android:id="@+id/sheet_item_title"
         style="@style/TextAppearance.FluentUI.PersistentBottomSheetHorizontalItem"
+        android:textColor="@color/persistent_bottom_sheet_horizontal_item"
```

Comment:
I think textColor is already part of style right?
If not,  please move it to style and we can remove this from here

Reply:
Will do